### ISSUE TITLE
Whether the Figure 2 is wrong in paper

### ISSUE THREAD

**Issue by lesleyhaha** [Microsoft/singleshotpose#69]
The c(x) in the function and the figure is not corresponding. In the code, conf = torch.exp(sharpness*(1 - dist/distthresh))-1. I think whether the c(x)(if D_T(x)<d_{th}) in paper need to minus 1, and the alpha=ln2 which corresponds to figure 2.

**Comment by btekin** [comment]
Fig. 2 is actually not wrong, but Eq. 1 in the paper should be normalized such that the highest confidence value is 1, as was also done in the code.

**Comment by buaadf** [comment]
I think Figure 2 is correct but Eq1 is wrong
